### PR TITLE
🔧 chore: Add meta tags to various pages

### PR DESCRIPTION
**Pull request by bryantanjw**
Adds `<Meta />` tags to various. from [86888ayxt](https://app.clickup.com/t/86888ayxt)

### CODE REVIEW

File: src/pages/generate/index.tsx [hunk 1]
```diff
@@ -57,6 +58,8 @@ export default function GeneratePage() {
   // desktop view
   return (
     <GenerationProvider>
+      <Meta title="Generate" description="Create and manage your generated images" deIndex />
```
Comment (fxisco) [inline src/pages/generate/index.tsx:61]: do we still need a description even if is a `deIndex`?

Comment (bryantanjw) [reply src/pages/generate/index.tsx:61]: good point. removing!

File: src/pages/collections/[collectionId]/index.tsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ export const getServerSideProps = createServerSideProps({
 export default function Collections({ collectionId }: { collectionId: number }) {
   return (
     <CollectionsLayout>
+      <Meta title={`Collection | ${collectionId}`} description="View the collection details" />
```
Comment (manuelurenah) [inline src/pages/collections/[collectionId]/index.tsx:34]: This Meta would be replaced with the one inside the `Collection` component used down below, no need for it here
```suggestion
```
We might need to truncate the description on that Meta though, can you please update it accordingly?

It should be something like the following
```jsx
<Meta
  // ...
  description={truncate(removeTags(collection.description), { length: 150 })}
/>
```

File: src/pages/user/[username]/index.tsx [hunk 3]
```diff
@@ -81,6 +82,8 @@ function ProfileOverview() {
 
   return (
     <>
+      <Meta title={`User Profile | ${user.username}`} description="View user profile and content" />
```
Comment (manuelurenah) [inline src/pages/user/[username]/index.tsx:85]: I think this is already handled inside the `ProfileLayout`, please double check this is not overwriting

**Review by manuelurenah — APPROVED** [review]

**Review by lrojas94 — APPROVED** [review]

**Pull request merged**